test/performance/dev.md

```diff
@@ -51,8 +53,8 @@ Take `dataplane-probe` benchmark for example:
 1. Patch istio:
 
    ```shell
-   kubectl patch hpa -n istio-system istio-ingressgateway \
-     --patch '{"spec": {"minReplicas": 10, "maxReplicas": 10}}'
+   kubectl patch deploy -n istio-system istio-ingressgateway \
+     --patch '{"spec": {"replicas": 10}}'
```

Comment:
won't HPA override this value?

Reply:
If following https://github.com/knative/serving/blob/master/DEVELOPMENT.md, `istio-minimal.yaml` is installed. There's no HPA for instio-ingressgateay.